conf.d/04-wallets

```diff
@@ -47,6 +47,14 @@ python setup.py install
 cd -
 rm -r trezor-*
 
+# install electrum dependency python-typing
+tar -zxf typing-*.tar.gz
+rm typing-*.tar.gz
+mv typing-*/ $SHARE/python-typing/
+cd $SHARE/python-typing
+sudo python3 setup.py install
+cd $SRC
+
```

Comment:
`python-typing` is now bundled in Electrum v3.2.3, so we don't need to preinstall it anymore.